mkdocs_monorepo_plugin/parser.py

```diff
@@ -199,10 +202,17 @@ def _prependAliasToNavLinks(self, alias, nav):
                         "[mkdocs-monorepo] We currently do not support nested !include statements inside of Mkdocs.")
                     raise SystemExit(1)
 
-                if key is None:
-                    nav[index] = "{}/{}".format(alias, value)
+                # Testing value starts with any of protocolSet
+                if (((value.split("//", 1))[0] + "//") in protocolSet):
```

Comment:
If we pretending start simple just looking for `http://` or `https://`, how about using `startswith` to make our code more readable?
```python
def format_nav_link(link):
 protocols = set(['http://', 'https://'])
 # true if the value is "external" link
 if (value.startswith(tuple(protocols))):
   return "{}".format(value)
 else:
   return "{}/{}".format(alias, value)

if key is None:
  nav[index] = format_nav_link(value)
else:
  nav[index][key] = format_nav_link(value)
```

But if we want a more robust validation, which checks if it's an external link and still a valid link, maybe we should decide to use a lib that validates URLs or even something built-in that's a little more comprehensive like:

```python
if (bool(urlparse(value).netloc)):
```

Reply:
mkdocs uses `urlsplit()` and checks for `scheme` or `netloc` to identify external links in nav. 

See [here](https://github.com/mkdocs/mkdocs/blob/fd0e9dedd27e4cb628ab98ff2723165110b38ed2/mkdocs/structure/nav.py#L125).

I'd suggest you do the same for this plugin.

Reply:
This works fine for me:

```python
from urllib.parse import urlsplit

[...]

def _prependAliasToNavLinks(self, alias, nav):

[...]

    def formatNavLink(alias, value):
        scheme, netloc, path, query, fragment = urlsplit(value)
        # true if the value is an external link
        if scheme or netloc:
            return "{}".format(value)
        else:
            return "{}/{}".format(alias, value)
    
    if key is None:
        nav[index] = formatNavLink(alias, value)
    else:
        nav[index][key] = formatNavLink(alias, value)
```